flink/v1.20/flink/src/main/java/org/apache/iceberg/flink/FlinkCatalog.java

```diff
@@ -384,13 +395,6 @@ public void renameTable(ObjectPath tablePath, String newTableName, boolean ignor
   @Override
   public void createTable(ObjectPath tablePath, CatalogBaseTable table, boolean ignoreIfExists)
       throws CatalogException, TableAlreadyExistException {
-    if (Objects.equals(
```

Comment:
Do we have a way to check if we can get the catalog properties in another way if they are not directly set by the table options?

Reply:
Modified to using json as per our offline discussion, so brought this check back to not allow users to create table in iceberg catalog using connector.

Reply:
Thanks!